spec/amp-actions-and-events.md

```diff
@@ -177,9 +177,9 @@ event.value</pre>
   </tr>
   <tr>
     <td><code>slideChange</code></td>
-    <td>Fired when the carousel's current slide changes.</td>
-    <td><pre>// Slide number.
-event.index</pre></td>
+    <td>Fired when the carousel's current slide changes and whether this is due to user interaction (not autoplay).</td>
+    <td><pre>event.index
+    event.userInteracted</pre></td>
```

Comment:
Do we need this to be part of actions/events API?

Reply:
Removed from the documentation and the AMP action that is fired.